src/Util/Lifecycle/Method/PayLaterMethodData.php

```diff
@@ -25,6 +25,21 @@ class PayLaterMethodData extends AbstractMethodData implements CheckoutDataMetho
 
     public const PAYPAL_PAY_LATER_FIELD_DATA_EXTENSION_ID = 'payPalPayLaterFieldData';
 
+    /**
+     * @var array<string, array<string, string|float>>
```

Comment:
```suggestion
     * @var array<string, array{currency: string, minAmount: float, maxAmount: float}>
```

with this, the array with the object like shape is described more precise. I am not sure, if there is a official way to annotate constants, but it should help PHPStan to understand the purpose better